CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Upcoming
-
+* When bumping to v0.0.6, also bump `bytes` to `1.0`
```

Comment:
Maybe shorten to "Bump `bytes` to 1.0" ? The version is implicit - it'll be underneath the 0.0.6 header once it releases.